libipl/libipl.H

```diff
@@ -31,6 +31,7 @@ enum ipl_error_type {
 	IPL_ERR_FSI_TGT_NOT_FOUND,
 	IPL_ERR_FSI_REG,
 	IPL_ERR_PIB_TGT_NOT_FOUND,
+	IPL_ERR_CLK_FAILED,
```

Comment:
Commit message:
- Looks like the sub-points alignment is not correct, please check it.
- Don't use the system local path of the device tree which you tested in the commit message
- JS for Commit message header, `libipl: p10: Init and Check the clock status`?

Reply:
reset doesnt indicate the complete changes.

I think i will put libipl: Clock initialization support for p10 based system